share/completions/elm.fish

```diff
@@ -0,0 +1,24 @@
+set -l commands repl init reactor make install bump diff publish
+
+complete -c elm -f
+# repl completions
+complete -c elm -n "not __fish_seen_subcommand_from $commands" -a repl -d 'Open up an interactive programming session'
+complete -c elm -n "__fish_seen_subcommand_from repl" -l no-colors -d 'Turn off the colors in REPL'
+complete -c elm -n "__fish_seen_subcommand_from repl" -l interpreter= -d 'Path to an alternative JS interpreter'
```

Comment:
I would rather put `"__fish_seen_subcommand_from repl"` in a some variable like so: `set -l repl_condition "__fish_seen_subcommand_from repl"`.